sdk/workloads/Azure.ResourceManager.Workloads/tests/Tests/SviCRUDTests.cs

```diff
@@ -0,0 +1,210 @@
+                    data: SviInstallPayload);
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine(ex);
+            }
+
+            var sviObject1 = await rg.GetSapVirtualInstanceAsync(SviName);
+
+            // Delete RG
+            await rg.DeleteAsync(WaitUntil.Completed);
+        }
+
+        [TestCase]
+        [RecordedTest]
+        public async Task TestSVIStartStopOperations()
+        {
+            string SviName = "C11";
+
+            SubscriptionResource subscription = await Client.GetDefaultSubscriptionAsync();
+            var lro = await subscription.GetResourceGroups().GetAsync("E2E-SVI-DevBox-27Jun-avzone-SLES-SAP-12-sp4-gen2");
+            ResourceGroupResource rg = lro.Value;
+
+            // Stop operation
+            var sviObject1 = await lro.Value.GetSapVirtualInstanceAsync(SviName);
+            await sviObject1.Value.StopAsync(WaitUntil.Completed);
+
+            var sviObjectstatus = await lro.Value.GetSapVirtualInstanceAsync(SviName);
+            Assert.AreEqual(sviObjectstatus.Value.Data.Status, SapVirtualInstanceStatus.Offline);
+
+            // Start operation
+            await sviObject1.Value.StartAsync(WaitUntil.Completed);
+            sviObjectstatus = await lro.Value.GetSapVirtualInstanceAsync(SviName);
+            Assert.AreEqual(sviObjectstatus.Value.Data.Status, SapVirtualInstanceStatus.Running);
+        }
+
+        /// <summary>
+        /// Gets an object of SAP Workload Resource for Single Server that should be used for PUT call.
+        /// </summary>
+        /// <param name="infraRgName"> Contains the infra RG
+        /// used as app resource group. </param>
+        /// <returns>PHP Workload Resource.</returns>
+        private SapVirtualInstanceData GetSingleServerPayloadToPut(string infraRgName, bool isInstall)
+        {
+            var testSapWorkloadJson = File.ReadAllText(Path.Combine(Directory.GetCurrentDirectory(), @"tests\SingleServerInstall.json"));
+            var deploymentConfigurationJson = JsonDocument.Parse(testSapWorkloadJson).RootElement;
+            DeploymentConfiguration deploymentConfiguration = DeploymentConfiguration.DeserializeDeploymentConfiguration(
+                deploymentConfigurationJson.GetProperty("properties").GetProperty("configuration"));
+
+            deploymentConfiguration.InfrastructureConfiguration.AppResourceGroup = infraRgName;
+
+            if (isInstall)
+            {
+                // Creating the installation payload after fetching the SshPrivateKey
+                ServiceInitiatedSoftwareConfiguration softwareConfiguration =
+                    ServiceInitiatedSoftwareConfiguration.DeserializeServiceInitiatedSoftwareConfiguration(
+                        deploymentConfigurationJson.GetProperty("properties").GetProperty("configuration").GetProperty("softwareConfiguration"));
+                softwareConfiguration.SshPrivateKey = this.GetKeyVaultSecret("E2ETestPrivatesshkey");
```

Comment:
let's remove this method (GetKeyVaultSecret and GetCertificateBySubjectName), and keyvault related imports.

User can add a ssh key to be used for installing in a file like json payload. let's read from that directly (similar to how i suggested to do for ssh key inside config).

Also, remember to checkin empty ssh key files, those have to be populated during test runs locally only.

Create a project level readme to add details on what all needs to be done to run this.

Reply:
Removed, and SetupInstructions added.